plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CQLSH_HOST=${CQLSH_HOST:-"cassandra"}
+CASSANDRA_WAIT_TIMEOUT=${CASSANDRA_WAIT_TIMEOUT:-"60"}
+DATACENTER=${DATACENTER:-"openshift"}
+KEYSPACE=${KEYSPACE:-"jaeger_v1_${DATACENTER}"}
+
+total_wait=0
+while true
+do
+  ping -c 1 ${CQLSH_HOST} > /dev/null 2>&1
```

Comment:
is this the typical way of checking that the container app is running? Seems line it might succeed as soon as the container / IP is available, but the Cassandra server may still be starting. 

@black-adder what did we use for e2e test? Didn't we wait for the actual response from Cassandra?